src/mibase.ts

```diff
@@ -748,3 +752,34 @@ function prettyStringArray(strings) {
 			return JSON.stringify(strings);
 	} else return strings;
 }
+
+async function cleanInvalidSocketPath(socketlists:string){
+	return new Promise((resolve, reject) => {
+			fs.readdir(socketlists, (err, files) => {
+				if (!err) {
+					if (files.length == 0) resolve('');
+					files.forEach((file)=>{
+						try {
+							const conn = net.connect(systemPath.join(socketlists, file));
```

Comment:
unclosed connection?

Reply:
The only thing needed now (manually): testing that everything works fine with two separate debugging sessions.
To easily test that outside of a dev environment it would be nice if the CI could create a vsix artifact...